_includes/landing-page/landing-description.html

```diff
@@ -0,0 +1,68 @@
+                From there you can take the OpenFaaS workshop, a series of tried-and-tested
+                self-paced labs which teach you everything you need to know about functions — and more.
+            </h1>
+        </div>
+        <div class="column is-12-tablet is-5-desktop is-offset-1-desktop">
+            <div class="columns is-multiline">
+                <div class="column is-12 is-spaced-bottom">
+                    <div class="columns">
+                        <div class="column is-3-tablet is-narrow-desktop has-text-centered-mobile">
+                            <p><img src="{{ site.baseurl }}/images/openfaas/store.png"></p>
+                        </div>
+                        <div class="column">
+                            <h1 class="title is-size-4 has-text-grey-darker">
+                                Functions Store
+                            </h1>
+                            <h1 class="title has-text-weight-light is-size-5-tablet is-size-4 has-text-grey">
+                                An ecosystem for sharing, reusing andcollaborating on functions
+                            </h1>
+                        </div>
+                    </div>
+                </div>
+                <div class="column is-12 is-spaced-bottom">
+                    <div class="columns">
+                        <div class="column is-3-tablet is-narrow-desktop has-text-centered-mobile">
+                            <p><img src="{{ site.baseurl }}/images/openfaas/template.png"></p>
+                        </div>
+                        <div class="column">
+                            <h1 class="title is-size-4 has-text-grey-darker">
+                                Templating system
+                            </h1>
+                            <h1 class="title has-text-weight-light is-size-5-tablet is-size-4 has-text-grey">
+                                Reduce boilerplate code, share code inthe templates store
+                            </h1>
+                        </div>
+                    </div>
+                </div>
+                <div class="column is-12 is-spaced-bottom">
+                    <div class="columns">
+                        <div class="column is-3-tablet is-narrow-desktop has-text-centered-mobile">
+                            <p><img src="{{ site.baseurl }}/images/openfaas/functions.png"></p>
+                        </div>
+                        <div class="column">
+                            <h1 class="title is-size-4 has-text-grey-darker">
+                                Functions or Microservices
+                            </h1>
+                            <h1 class="title has-text-weight-light is-size-5-tablet is-size-4 has-text-grey">
+                                Deploy existing microservices using Express.js,Sinatra, ASP.NET Core or simplify with functions.
```

Comment:
Missing space after the comma before `Sinatra`

![Screenshot from 2019-04-17 19-53-37](https://user-images.githubusercontent.com/12687192/56330291-070a4380-614d-11e9-8169-b9c6d8be1946.png)